webui/react/playwright.config.ts

```diff
@@ -6,6 +6,15 @@ import { defineConfig, devices } from '@playwright/test';
 import * as dotenv from 'dotenv';
 dotenv.config();
 
+const serverAddess = process.env.PW_SERVER_ADDRESS
+if (serverAddess === undefined) {
+  throw new Error('Expected PW_SERVER_ADDRESS to be set.')
+}
+const port = Number(new URL(serverAddess).port)
```

Comment:
one more edge case,  
idk if its clean but 

```suggestion
const port = Number(new URL(serverAddess).port || 'not exist')
```